velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -615,6 +616,30 @@ TEST_F(E2EFilterTest, combineRowGroup) {
   EXPECT_EQ(parquetReader.numberOfRows(), 5);
 }
 
+TEST_F(E2EFilterTest, configurableWriteSchema) {
+  rowType_ = ROW({"c0"}, {INTEGER()});
+
+  std::unique_ptr<test::VectorMaker> vectorMaker_{
+      std::make_unique<test::VectorMaker>(leafPool_.get())};
+  std::vector<RowVectorPtr> batches;
+  for (int i = 0; i < 5; i++) {
+    auto data = vectorMaker_->rowVector({vectorMaker_->flatVector<int32_t>(
+        20'000, [](auto row) { return row * 10; })});
```

Comment:
Do you need to make large vectors? Can you not use 100 row vectors instead? The test will run faster.